Guided Hunting - Detect potential network beaconing using Apache Spark via Azure Synapse.ipynb

```diff
@@ -0,0 +1,1451 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Guided Hunting - Detect potential network beaconing using Apache Spark via Azure Synapse\r\n",
+        "\r\n",
+        "__Notebook Version:__ 1.0<br>\r\n",
+        "__Python Version:__ Python 3.8 - AzureML<br>\r\n",
+        "__Required Packages:__ azureml-synapse, Msticpy, azure-storage-file-datalake <br>\r\n",
+        "__Platforms Supported:__  Azure Machine Learning Notebooks connected to Azure Synapse Workspace\r\n",
+        "     \r\n",
+        "__Data Source Required:__ Yes\r\n",
+        "\r\n",
+        "__Data Source:__ CommonSecurityLogs\r\n",
+        "\r\n",
+        "__Spark Version:__ 3.1 or above\r\n",
+        "    \r\n",
+        "### Description\r\n",
+        "In this sample guided scenario notebook, we will demonstrate how to set up continuous data pipeline to store data into azure data lake storage (ADLS) and \r\n",
+        "then hunt on that data at scale using distributed processing via Azure Synapse workspace connected to serverless Spark pool. \r\n",
+        "Once historical dataset is available in ADLS , we can start performing common hunt operations, create a baseline of normal behavior using PySpark API , Azure Sentinel watchlists and also apply data transformations \r\n",
```

Comment:
Please remove "Azure Sentinel watchlists" so users won't expect it in this version